src/main/java/racingcar/controller/RacingCarController.java

```diff
@@ -0,0 +1,80 @@
+package racingcar.controller;
+
+import java.util.List;
+import racingcar.dto.RacingCarNamesRequest;
+import racingcar.dto.RacingCarStatusResponse;
+import racingcar.dto.RacingCarWinnerResponse;
+import racingcar.dto.TryCountRequest;
+import racingcar.service.RacingCarService;
+import racingcar.service.RandomMoveStrategy;
+import racingcar.service.TryCount;
+import racingcar.view.RacingCarView;
+
+public class RacingCarController {
+    private final RacingCarService racingCarService;
+    private final RacingCarView racingCarView;
+
+    public RacingCarController(RacingCarService racingCarService, RacingCarView racingCarView) {
+        this.racingCarService = racingCarService;
+        this.racingCarView = racingCarView;
+    }
+
+    public void start() {
+        createCar();
+        TryCount tryCount = new TryCount(getTryCount());
+        playGame(tryCount);
+    }
+
+    private void createCar() {
+        RacingCarNamesRequest racingCarNamesRequest = receiveCarNames();
+        racingCarService.createCars(racingCarNamesRequest);
+    }
+
+    private RacingCarNamesRequest receiveCarNames() {
+        try {
+            return racingCarView.receiveCarNames();
+        } catch (RuntimeException e) {
+            racingCarView.printExceptionMessage(e);
+            return receiveCarNames();
+        }
+    }
+
+    private int getTryCount() {
+        TryCountRequest tryCountRequest = receiveTryCount();
+        return tryCountRequest.getTryCount();
+    }
+
+    private TryCountRequest receiveTryCount() {
+        try {
+            return racingCarView.receiveTryCount();
+        } catch (RuntimeException e) {
+            racingCarView.printExceptionMessage(e);
+            return receiveTryCount();
+        }
+    }
+
+    private void playGame(TryCount tryCount) {
+        RandomMoveStrategy randomMoveStrategy = new RandomMoveStrategy();
+        racingCarView.printStartMessage();
+        for (int i = 0; i < tryCount.getCount(); i++) {
+            racingCarService.moveCars(randomMoveStrategy);
+            printCarStatuses();
+        }
```

Comment:
```suggestion
        while (tryCount.isAvailable()) {
            ...
        }
```

검증을 위해 원시값을 포장한 VO로만 사용할 수도 있겠지만, 
자동차 경주 반복을 물어볼 수 있는 도메인 객체가 될수도 있겠네요.

Reply:
오오... 확실히 VO보다 책임을 가지고 있는 도메인 객체로서의 역할이 더 어울려 보이네요!